manifest_staging/charts/csi-secrets-store-provider-azure/templates/podsecuritypolicy.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.rbac.pspEnabled }}
+{{- if and .Values.rbac.pspEnabled (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy") }}
```

Comment:
Do we want to support this till 1.24 EOL?

Reply:
We're supporting it with this change. We're checking if the cluster still has the capability, and if yes, we're deploying the pod security policy.

Reply:
I asked the wrong question... 😅. But I got the answer. After 1.24 EOL we can completely remove this. Thanks.